Documentation/LanguageServer/MinGW.md

```diff
@@ -1,6 +1,43 @@
 For developers using MinGW on Windows, we recommend you start with the following **c_cpp_properties.json** template.  Select "C/Cpp: Edit Configurations" from the command palette to create this file if you haven't already.
 
-Note that you may have to change the MinGW version number to match what you have installed. Eg. `C:/MinGW/lib/gcc/mingw32/5.3.0/` instead of `C:/MinGW/lib/gcc/mingw32/6.3.0/`.
+## With extension version 0.16.1 and higher:
+
+Starting with version 0.16.1, if you set `compilerPath` and `intelliSenseMode`, you no longer need to copy the system include path or defines to `includePath` and `defines` to enable IntelliSense to work properly. However, `browse.path` still needs to be updated manually to add the system include paths to enable code browsing for system headers. For example:
```

Comment:
Why are you mentioning intelliSenseMode? It doesn't actually need to be set if the OS/compiler match.

Reply:
The extension always sets this, and it doesn't look like the query compiler message returns a value back for this.  Do we override it if it's wrong?

Reply:
Oh, you're trying to say the user needs to set the intelliSenseMode to match the compilerPath compiler if there's a mismatch (e.g. msvc-x64 with a gcc/clang compiler).

Reply:
yes.  I'm just trying to call out that property since some users weren't changing it to `clang-x64` and it defaults to `msvc-x64` on Windows.

Reply:
I might be able to word this better